### PR TITLE
guix: Silence `getent(1)` invocation, doc fixups

### PR DESCRIPTION
**Pull request by dongcarl**
Otherwise the `getent(1)` checks will print out the default http, https, and ftp ports, making it seem like something is being spawned that is listening on those ports, which is not the case.

### CODE REVIEW

File: contrib/guix/guix-build [hunk 1]
```diff
@@ -190,7 +190,7 @@ fi
 # Services database must have basic entries
 ################
 
-if ! getent services http https ftp; then
+if ! getent services http https ftp > /dev/null 2>&1; then
```
Comment (LarryRuane) [inline contrib/guix/guix-build:193]: utACK 3c4d2c418e62d421a8bd7640ef5384251c892406
```suggestion
if ! getent services http https ftp &>/dev/null
```

**Review by LarryRuane — APPROVED** [review]

**Comment by laanwj** [comment]
ACK 3c4d2c418e62d421a8bd7640ef5384251c892406 thanks for addressing this

**Review by theStack — APPROVED** [review]
ACK 3c4d2c418e62d421a8bd7640ef5384251c892406

Didn't run the script but verified the behaviour in the shell:
```
$ getent services http https ftp; echo $?
http                  80/tcp www
https                 443/tcp
ftp                   21/tcp
0
$ getent services http https ftp unknown-service; echo $?
http                  80/tcp www
https                 443/tcp
ftp                   21/tcp
2
```

PR branch:
```
$ getent services http https ftp > /dev/null 2>&1; echo $?
0
$ getent services http https ftp unknown-service > /dev/null 2>&1; echo $?
2
```

**Comment by dongcarl** [comment]
Maintainers: would you mind if I tacked on a few README changes to this PR? I'm thinking of addressing the posthumous comments here:
- https://github.com/bitcoin/bitcoin/pull/21711#pullrequestreview-710604005
- https://github.com/bitcoin/bitcoin/pull/21711#pullrequestreview-710760594

**Comment by fanquake** [comment]
@dongcarl sgtm

**Review by LarryRuane — COMMENTED** [review]
utACK f0e00d0173087b29e0a1045a4073a292c55bd954
As long as you're changing `INSTALL.md`, if you retouch (I'll reack right away), there are a couple other typos:
- s/has a known problems/has known problems/
- s/erratas/errata/

**Comment by jonatack** [comment]
See #22527 for more tidy-ups to pull into this PR? The title of this PR could be changed, as it wasn't clear to me that these were being addressed.

**Comment by sipa** [comment]
utACK f0e00d0173087b29e0a1045a4073a292c55bd954. The current output is rather confusing.

**Review by fanquake — APPROVED** [review]
ACK a884a1edcd1b795799f1be65df17462041741dc3

**Comment by MarcoFalke** [comment]
Backported in #22534

**Pull request merged**